lib/components/tooltip.js

```diff
@@ -64,9 +66,17 @@ const addWebsiteLink = (data) => {
     return undefined;
   }
 
+  let iconClass = 'icon icon-globe'
+
+  if(isGithubUrl(data.homepage)) {
+    iconClass += ' github-icon'
+  } else if (isGitlabUrl(data.homepage)) {
+    iconClass = ' gitlab-icon'
+  }
+
   const icon = el('i');
   setAttr(icon, {
-    className: 'icon icon-globe',
+    className: iconClass,
```

Comment:
Once you extract the logic for adding the `className` into a function as I described, this line can be changed to
```js
setAttr(icon, {
  className: getRepoIcon(data.homepage),
});
```